docs/mathematics/matrix-addition.mdx

```diff
@@ -0,0 +1,172 @@
+---
+title: Matrix Addition
+slug: /mathematics/matrix-addition
+---
+## Introduction
+
+In mathematics, matrix addition is the operation of adding two matrices by adding the corresponding entries together. Two matrices must have an equal number of rows and columns to be added.
+
+## Explanation
+
+Consider two matrices `A` and `B`, the sum of two matrices `A` and `B` will be a matrix that has the same number of rows and columns as `A` and `B`. The sum of `A` and `B` denoted `A` + `B`, is computed by adding corresponding elements of `A` and `B`
+ 
+For Example 
+
+![image](https://i.postimg.cc/Jn8yyrzg/image.png)
+
+## Algorithm
+
+- Input matrix `A` and matrix `B`
+- Input matrix `A`
+- Input matrix `B` 
+- If the number of rows and number of columns of matrix `A` and matrix `B` is equal
+- loop from `i`=0 to number of rows in A do
+- loop from `j`=0 to number of rows in A do 
+- `Result[i,j]` =  `A[i,j]` + `B[i,j]`
+- Display `Result`
+
+
+## Code
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+<Tabs
+  defaultValue="cpp"
+  values={[
+    { label: 'C++', value: 'cpp', },
+    { label: 'Python', value: 'py', },
+  ]
+}>
+
+<TabItem value="cpp">
+
+```cpp
+// C++ program to implement Matrix Addition
+#include <bits/stdc++.h>
+using namespace std;
+
+int main()
```

Comment:
Don't put the entire logic in the `main()` function. Use `main()` only for the driver code.